rpcclient/examples/customcommand/main.go

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2014-2017 The btcsuite developers
+// Copyright (c) 2019-2020 The Namecoin developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
```

Comment:
Not sure about this copyright notice. Should prob. follow the other files in this project?

Reply:
Since some of the code was copied from Namecoin, I'm not certain that it's legal to strip the Namecoin attribution.  Is there a different form that you'd prefer?

Reply:
If you added namecoin code (and it is ISC of course) then you do need to keep their copyright in there.

Reply:
Just verified that the license is ISC, so we need to keep the notice.